sealink/quickprint
# PLAT-28 Migrate to Github Action from Travis

**WHY**
Migrate to Github Action from Travis CI.

**How to test:**
Check the [test build](https://github.com/sealink/quickprint/runs/3480474122?check_suite_focus=true) in the Action Workflow.

Current build fails because of the [PR](https://github.com/sealink/ticket_printer/pull/21) for `ticket_print` not merged/release yet.
